scripts/generate_parser_test_files.py

```diff
@@ -123,11 +123,15 @@ def main() -> None:
             for html in html_mapping.values():
                 versioned_parser = html.publisher.parser(html.crawl_date)
                 extraction = versioned_parser.parse(html.content)
-                new = {attr: value for attr, value in extraction.items() if attr in arguments.attributes}
+                missing_attributes = set(arguments.attributes) - set(
+                    test_data.get(type(versioned_parser).__name__) or {}
```

Comment:
If I'm not mistaken, this line is only needed to update old test cases. Shouldn't it be removed as soon as all old test data is updated?

Reply:
Depends if we want to add additional attributes in the future ;)

Reply:
True :)